src/Compilers/CSharp/Test/Syntax/Parsing/UsingDirectiveParsingTests.cs

```diff
@@ -4077,4 +4037,1252 @@ public void TestMakeref()
         }
         EOF();
     }
+
+    [Fact]
+    public void TestUnsafeStatic1_CSharp11_NoUnsafeFlag()
+    {
+        var text = @"using unsafe static System.Console;";
+
+        CreateCompilation(text, parseOptions: TestOptions.Regular11).VerifyDiagnostics(
+            // (1,1): hidden CS8019: Unnecessary using directive.
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using unsafe static System.Console;").WithLocation(1, 1),
+            // (1,7): error CS8652: The feature 'using type alias' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.ERR_FeatureInPreview, "unsafe").WithArguments("using type alias").WithLocation(1, 7),
+            // (1,14): error CS9133: 'static' modifier must precede 'unsafe' modifier.
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.ERR_BadStaticAfterUnsafe, "static").WithLocation(1, 14));
+
+        UsingTree(text,
+            // (1,14): error CS9133: 'static' modifier must precede 'unsafe' modifier.
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.ERR_BadStaticAfterUnsafe, "static").WithLocation(1, 14));
+
+        N(SyntaxKind.CompilationUnit);
+        {
+            N(SyntaxKind.UsingDirective);
+            {
+                N(SyntaxKind.UsingKeyword);
+                M(SyntaxKind.StaticKeyword);
+                N(SyntaxKind.UnsafeKeyword);
+                N(SyntaxKind.QualifiedName);
+                {
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "System");
+                    }
+                    N(SyntaxKind.DotToken);
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "Console");
+                    }
+                }
+                N(SyntaxKind.SemicolonToken);
+            }
+            N(SyntaxKind.EndOfFileToken);
+        }
+        EOF();
+    }
+
+    [Fact]
+    public void TestUnsafeStatic1_CSharp11_UnsafeFlag()
+    {
+        var text = @"using unsafe static System.Console;";
+
+        CreateCompilation(text, options: TestOptions.UnsafeDebugDll, parseOptions: TestOptions.Regular11).VerifyDiagnostics(
```

Comment:
>TestOptions.UnsafeDebugDll [](http://example.com/codeflow?start=41&length=26)

The `UnsafeDebugDll` option does not affect parsing, so I would either remove the check on syntax tree below or move the CreateCompilation call with `UnsafeDebugDll` to test above.
This comment likely applies to other tests below as well.

Reply:
not sure what you mean here.  the point here is to both check syntax errors (done with UsingTree) as well as what errors are semantic (teh CreateCompilation case).  I don't want to hten have a pointless "you need /unsafe flag" error in the semantic check.

Reply:
I was suggesting either: 
1. removing the `UsingTree` from `TestUnsafeStatic1_CSharp11_UnsafeFlag`. The parse tree is already verified by `TestUnsafeStatic1_CSharp11_NoUnsafeFlag` and the `unsafe` flag doesn't affect parse tree.
2. removing the `TestUnsafeStatic1_CSharp11_UnsafeFlag` test and moving it's `CreateCompilation`/`VerifyDiagnostics` checks into the test above (and tweaking that test's name)

Added "nit:" marker, so this is not blocking.